src/query.js

```diff
@@ -77,6 +81,17 @@ class Query {
     return result;
   }
 
+  find(fn) {
+    let iterator = this.generator();
+    let result;
+    for (let next = iterator.next(); !next.done; next = iterator.next()) {
+      if (fn(next.value)) {
+        return result = next.value
```

Comment:
You don't need to assign result here. You can just return next.value

Reply:
made the changes. thank you.